WIKI.org

```diff
@@ -137,7 +135,8 @@ to your Dropbox, here are your options:
  have HTTP ok status.
  #+END_EXAMPLE
 
- then the =RewriteRule= in the above example is either missing from, or ineffective in your webserver config.
+ then the =RewriteRule= in the above example is either missing from,
```

Comment:
which `RewriteRule` in the above example? You deleted it.

Reply:
Ooops haha good spot! I went through several iterations of the config before settling on `<LimitExcept>`, and forgot to update this.

Reply:
Fixed.